src/nft/components/explore/Banner.tsx

```diff
@@ -155,8 +166,17 @@ const Banner = () => {
       ) : null}
       <BannerMainArea>
         <HeaderContainer>
-          Better prices. {!isMobile && <br />}
-          More listings.
+          {isMobile ? (
+            <>
+              Better prices. <br />
+              Every listing.
+            </>
+          ) : (
+            <>
+              Better prices. <br />
+              More listings.
```

Comment:
this was unclear from the ticket, the text should stay the same but on mobile it should be on 2 lines

Reply:
Might be worth checking with the product, because Figma does have two different copies on mobile/desktop.

Reply:
I'll follow-up and let you know

Reply:
sounds like a mistake in Figma